dashboard/src/actions/overviewActions.js

```diff
@@ -109,7 +107,7 @@ export const getMetaDataActions =
   (dataset, actionType, actionValue) => async (dispatch) => {
     const method = metaDataActions[actionType];
     const payload = { [method]: actionValue };
-    return dispatch(updateDataset(dataset, payload));
+    dispatch(updateDataset(dataset, payload));
   };
```

Comment:
I keep getting lost in all the levels of redirection, here; and, I don't remember what the effect of _not_ returning the promise from `dispatch()` would be, here.

However, at line 216 we're going call `getMetaDataActions()` and pass whatever it returns to `dispatch()`.  The code here is going to return a callable which the caller's call to `dispatch()` will cause to be invoked at some point later.  When it's called it's going to do some small bits of setup and then it's going to pass what `updateDataset()` returns off to be dispatched.  So, we're still dispatching a dispatch, which seems...overdone.

Why can't `getMetadataActions()` simply call `updateDataset()` and return whatever it returns?  The caller of `getMetadataActions()` will then pass _that_ to `dispatch()`, and we'll only end up dispatching things once.